_includes/_sidebar.html

```diff
@@ -3,11 +3,9 @@
         <h3><a href="/events/">ISC Events</a></h3>
         
 	<p>
-		The InnerSource Commons APAC 2020 Virtual Summit is taking place on the 2nd and 3rd of December 2020 - <a href="https://eventyay.com/e/3dbaaa50">register today</a>!
-	  <a href="/events/isc-apac-dec-2020">Click here</a> to find out more!
+    See you in 2021 Event Season.
 	</p>
-
-        <p>Sessions from the InnerSource Commons Fall 2020 Virtual Summit are <a href="https://www.youtube.com/playlist?list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy">available.</p>
+  <p>Sessions from the InnerSource Commons Fall 2020 Virtual Summit are <a href="https://www.youtube.com/playlist?list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy">available.</p>
```

Comment:
Fixed HTML. Also made it clear that the videos are on YouTube.

```suggestion
  <p>Sessions from the InnerSource Commons Fall 2020 Virtual Summit are <a href="https://www.youtube.com/playlist?list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy">available on YouTube.</a></p>
```